### PR TITLE
Translate Frontend Parse modal

### PR DESCRIPTION
**Pull request by stevietv**
#### Database Migration
NO

#### Description
Translate Frontend Parse modal further than it already was

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: frontend/src/Parse/ParseResult.tsx [hunk 1]
```diff
@@ -228,7 +228,13 @@ function ParseResult(props: ParseResultProps) {
 
         <ParseResultItem
           title={translate('CustomFormats')}
-          data={<EpisodeFormats formats={customFormats} />}
+          data={
+            customFormats && customFormats.length > 0 ? (
```
Comment (markus101) [inline frontend/src/Parse/ParseResult.tsx:232]:
```suggestion
            !!customFormats?.length ? (
```

or keep the `> 0` 🤷

**Review by markus101 — CHANGES_REQUESTED** [review]

File: frontend/src/Parse/ParseResult.tsx (hunk 1 above)
Comment (stevietv) [reply frontend/src/Parse/ParseResult.tsx:232]: lint doesn't like this:
```
  232:13  warning  Unexpected negated condition  no-negated-condition
  232:13  error    Redundant double negation     no-extra-boolean-cast
```

Comment (markus101) [reply frontend/src/Parse/ParseResult.tsx:232]: Ahh right, `customFormats?.length` should be sufficient

Comment (stevietv) [reply frontend/src/Parse/ParseResult.tsx]: yep

**Review by markus101 — APPROVED** [review]

**Pull request merged**